reference/configuration/framework.rst

```diff
@@ -2252,13 +2252,19 @@ Name of the workflow you want to create.
 audit_trail
 """""""""""
 
-**type**: ``array``
+**type**: ``bool``
```

Comment:
unsure of this one, why this one was array?

Reply:
Probably a copy+paste error.

Reply:
This is indeed a `bool`